configs/base.yml

```diff
@@ -22,5 +22,6 @@ MANAGER:
 
   MIGRATION_BACKEND: 'gloo'
   MIGRATION_CACHE_BLOCKS: 512
+  MIGRATION_INTERNAL_CACHE_SIZE: 1
```

Comment:
I recommend renaming these two parameters as `migration_buffer_blocks` and `migration_internal_buffer_num`
Namely, I prefer "buffer" than "cache" because this is actually a buffer not a cache; and the second parameter is the "number" of the buffers, not the size

Reply:
also, the word "cache" could be easily confused with kv cache

Reply:
I aggre with you